templates/CRM/Form/basicForm.tpl

```diff
@@ -7,7 +7,7 @@
  | and copyright information, see https://civicrm.org/licensing       |
  +--------------------------------------------------------------------+
 *}
-<div class="crm-block crm-form-block crm-{$formName}-block">
+<div class="crm-block crm-form-block crm-{$entityInClassFormat}-block">
```

Comment:
This tpl is only being used from a few admin forms - which do not seem to assign formName - but do assign entityInClassFormat

Reply:
![image](https://github.com/civicrm/civicrm-core/assets/336308/51ece291-ff7c-4c12-a2ff-febb8942286c)